rooms.js

```diff
@@ -708,22 +708,22 @@ class GlobalRoom {
 			// The server is in lockdown, the final battle has finished, and the option is set
 			// so we will now automatically kill the server here if it is not updating.
 			if (Chat.updateServerLock) {
-				this.notifyRooms(notifyPlaces, '<div class="broadcast-red"><b>Automatic server lockdown kill canceled.</b><br /><br />The server tried to automatically kill itself upon the final battle finishing, but the server was updating while trying to kill itself.</div>');
+				this.notifyRooms(notifyPlaces, '|html|<div class="broadcast-red"><b>Automatic server lockdown kill canceled.</b><br /><br />The server tried to automatically kill itself upon the final battle finishing, but the server was updating while trying to kill itself.</div>');
```

Comment:
You seem to be editing these lines without switching over to the new string standards.

For the record, the new string standard expects `` ` `` here.